packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -364,6 +364,19 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
     });
   }
 
+  @override
+  Future<void> animateCameraWithDuration(
+    CameraUpdate cameraUpdate, {
+    required int mapId,
+    Duration? duration,
```

Comment:
Do we need a whole new interface method to add an optional named parameter? Can we do this by adding `Duration? duration` to the `animateCamera` method?

Reply:
> Do we need a whole new interface method to add an optional named parameter?

Yep :(

https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-platform-interface-method-parameters